lib/stripe/transfer.rb

```diff
@@ -9,6 +9,8 @@ class Transfer < APIResource
 
     OBJECT_NAME = "transfer".freeze
 
+    custom_method :cancel, http_verb: :post
```

Comment:
We should add tests with mocking for this since stripe-mock does not support that endpoint

Reply:
👍